platform_umbrella/apps/verify/test/traditional_test.exs

```diff
@@ -0,0 +1,42 @@
+defmodule Verify.TraditionalTest do
+  use Verify.TestCase, async: false, batteries: ~w(traditional_services)a
+
+  require Logger
+
+  @new_traditional_path "/traditional_services/new"
+  @show_traditional_path ~r(/traditional_services/[\d\w-]+/show$)
+
+  @container_panel Query.css("#containers_panel-containers")
+
+  verify "can create traditional service", %{session: session} do
+    service_name = "int-test-#{:rand.uniform(10_000)}"
+
+    session
+    # create service
+    |> visit(@new_traditional_path)
+    |> assert_has(h3("New Traditional Service"))
+    |> fill_in_name("service[name]", service_name)
+    # add container
+    |> find(@container_panel, fn e -> click(e, Query.button("Add Container")) end)
+    |> fill_in(Query.text_field("container[name]"), with: "echo")
+    |> fill_in(Query.text_field("container[image]"), with: "ealen/echo-server:latest")
+    |> fill_in(Query.text_field("container[path]"), with: "")
```

Comment:
Why do we need to fill in empty? Does this mean users will need to do remove/put in empty themselves?

Reply:
No, I can probably remove that. It was copied over from knative and was there while I was troubleshooting. I'll double check and remove in the next PR.